projects/hello_world/src/hello_world.c

```diff
@@ -0,0 +1,35 @@
+//
+// Created by henry on 2021-01-09.
+//
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+void callback(SoftTimerId id, void *context) {
+  static bool b_increment = false;
+  Counters *Counter = context;
+  LOG_DEBUG("%d\n", ++Counter->counter_a);
+  if (b_increment) {
+    LOG_DEBUG("%d\n", ++Counter->counter_b);
```

Comment:
Using pre-incrementing is clever, but it also makes it a bit harder to read. Consider just incrementing after printing on another line.

Also it's technically not required I suppose, but to match the screenshot in the module, add `Counter A:` and `Counter B:` to the logging.